api/descriptions/policies/vulnerability_host_get.md

```diff
@@ -0,0 +1,16 @@
+Retrieves the vulnerability policy for your host deployments.
+A policy consists of ordered rules.
+
+This endpoint maps to the policy table in **Defend > Vulnerabilities > Hosts** in the Console UI.
```

Comment:
@KICKdesigns In this case, the exact path will be **Defend > Vulnerabilities > Hosts > Running hosts**.